lib/grpc/google/rpc.ex

```diff
@@ -0,0 +1,20 @@
+defmodule GRPC.Google.Rpc do
+  @moduledoc false
+
+  @spec encode_status(Google.Rpc.Status.t()) :: String.t()
+  def encode_status(%Google.Rpc.Status{} = status) do
```

Comment:
I rather isolate these two function around encoding and decoding the rpc status, just internal module for now